bundles/org.openhab.binding.miio/README.base.md

```diff
@@ -172,7 +172,13 @@ Firmware of the device don't accept commands coming from other subnets.
 Set the communication in the thing configuration to 'cloud'.
 
 _Cloud connectivity is not working_
-The most common problem is a wrong userId/password. Try to fix your userId/password.
+The most common problem is a wrong userId/password. Try to fix your userId/password. The login userId & password are entered in the binding config page.
+If the problem persists you can try the following:
+
+* Your ip might need to be validated/confirmed. Logon to https://account.xiaomi.com/ **from the ip of your openHAB server** with a browser.
```

Comment:
I do not understand the second sentence.

Reply:
I rephrased it. Hope it is more clear this time

Reply:
@jlaur : do you understand what means "Logon to https://account.xiaomi.com/ **from the ip of your openHAB server** with a browser."?

Reply:
I'm not sure I understand it, but let me explain how I understand it: Xiaomi need a kind of verification that a specific IP address is assigned to the user claiming it. Or, more specifically, the the IP address is currently assigned (dynamically or statically) by the ISP to the users router. So I assume we are talking about the public IP address. In this case I would assume that a browser on any computer within the LAN behind the router (assuming only one public IP address) can be used to perform this "validation". So I don't understand it like I would need to install and try to run some browser on my openHABian installation running on a Raspberry Pi.

In other words, back to my intro: No, I'm not sure I understand it, I would need some more context, and I think it could be further rewritten after that.

Reply:
@marcelrv : we have clearly understood that. What is not clear for us is the new instruction you are providing in the documentation: "Logon to https://account.xiaomi.com/ from the ip of your openHAB server with a browser.".
If it is not clear for both of us, I am afraid it will not be clear for many users. Only this sentence has to be clarified to be understandable what exactly has to be done.
I am sorry in case it is obvious for you.

Reply:
_If it is not clear for both of us, I am afraid it will not be clear for many users._ fully agree. 
the matter indeed is how to best describe this so it is clear to everyone

Reply:
Maybe a screen capture could help?

Reply:
I'll check with @xduseko if I can use his screenshot. I can't reproduce this myself

Reply:
The sentence I did not understand is still there, unchanged ; I am not sure the added screen capture is really helpful to understand it.
But ok, I will not insist more, let's assume the users will understand better than me what you mean.

Reply:
I did one more attempt to clarify it better, and trying to understand in which scenario's there might be issues.
The vast majority won't face this as their (external) ip is equal to their regular  pc/browser ip.

Reply:
Ok thank you, I finally understood.
Maybe you could replace "connect **from the ip of your openHAB server** with a browser" by "connect with your browser **while no VPN is enabled** (to not hide your external IP address)". WDYT ?

And the screen capture is no more necessary. WDYT ?

Reply:
Maybe you could replace "connect from the ip of your openHAB server with a browser" by "connect with your browser while no VPN is enabled (to not hide your external IP address)". WDYT ?
I think that is the most likely usecase indeed, but I think the user creating the issue actually was hosting his OH server somewhere else (on a VPS), why I understood he made a bridge to that server and was running a browser from the ip of his hosting provider. (He was by running some vpn docker with port forwarding, he also could have used some SSH tunneling I think) 
this is why I was emphasizing the ' from the ip of your openHAB server ' part... 

Do you think it is more clear if i change it to something like "browse with the same external ip address as your openHAB server to...http etc"

The screenshot does not contribute much, no problem removing it...

Reply:
> Do you think it is more clear if i change it to something like "browse with the same external ip address as your openHAB server to...http etc"

Yes, I believe